src/main/java/at/gammastrahlung/monopoly_server/game/Game.java

```diff
@@ -59,8 +59,8 @@ public class Game {
 
     @Getter
     private int currentPlayerIndex = 0;
-
-
+@Getter
+    private static List<Player> playerListForId = new ArrayList<>();
```

Comment:
What's the difference between this and players?
There is no reason to track the players twice